<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>450</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(70, 70, 70);</string>
   </property>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>50</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>genetic_algorithm_gui</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="selectFormulaLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>select_formula</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bestIndividualsLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>best_individuals</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fromEachPopulationLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>from_each_population</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="firstFormulaRadioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>425</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3 * pow(x, 2) + x * y + 2 * pow(y, 2) - x - 4 * y</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="secondFormulaRadioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>425</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
     </font>
    </property>
    <property name="text">
     <string>pow(y, 2) + 2 * x * y - 4 * x - 2 * y - 3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="thirdFormulaRadioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>425</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
     </font>
    </property>
    <property name="text">
     <string>-2 * pow(y, 2) - x - 14 * y + y * sqrt(x)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="fifthFormulaRadioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>425</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
     </font>
    </property>
    <property name="text">
     <string>4 + pow((pow(x, 2) + pow(y, 2)), 2/3.)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="fourthFormulaRadioButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>425</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
     </font>
    </property>
    <property name="text">
     <string>exp(x + y) * (pow(x, 2) - 2 * pow(y, 2))</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="listOfIndividualsTextBrowser">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>155</y>
      <width>243</width>
      <height>275</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(30, 30, 30);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SF Mono'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getResultButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: rgb(255, 255, 255);
background-color: rgb(50, 50, 50);
border: 1px solid(50, 50, 50);
}
QPushButton:pressed{
background-color: rgb(60, 60, 60);
}
</string>
    </property>
    <property name="text">
     <string>get_result</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>450</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SF Mono</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(30, 30, 30);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>43</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
